Apromore-Database/src/main/resources/db/migration/changeLog-Schema.yaml

```diff
@@ -1885,3 +1885,36 @@ databaseChangeLog:
                  constraints:
                    nullable: false
                  remarks: 'Indicator as to whether the attribute can be used as a perspective'
+
+ - changeSet:
+     id: 20220622090000
+     author: frankm
+     preConditions:
+       - onFail: MARK_RAN
+       - tableExists:
+           tableName: storage
+     changes:
+       - dropTable:
+           context: H2
+           cascadeConstraints:  true
+           tableName: storage
+
+ - changeSet:
+     id: 20220622100000
+     author: frankm
+     changes:
+       - sql:
+           dbms: 'h2'
+           splitStatements: true
+           sql: CREATE TABLE storage (id bigint NOT NULL AUTO_INCREMENT, storage_path varchar(200) NOT NULL, prefix varchar(100) NOT NULL, "key" varchar(200) DEFAULT NULL, created varchar(100) NOT NULL,updated varchar(100) NOT NULL,PRIMARY KEY (id));
+           stripComments: false
+
+       - addForeignKeyConstraint:
```

Comment:
storage_id is also a foreign key for process model version

line: 1230
changeset 20210922110000
```
       - addForeignKeyConstraint:
           baseColumnNames: storage_id
           baseTableName: process_model_version
           constraintName: fk_process_model_version_storage
           deferrable: false
           initiallyDeferred: false
           referencedColumnNames: id
           referencedTableName: storage
           validate: true
```